tests/behavior/list.rs

```diff
@@ -301,7 +305,11 @@ pub async fn test_remove_all(op: Operator) -> Result<()> {
         "x/", "x/y", "x/x/", "x/x/y", "x/x/x/", "x/x/x/y", "x/x/x/x/",
     ];
     for path in expected.iter() {
-        op.create(path).await?;
+        if path.ends_with('/') {
+            op.create_dir(path).await?;
+        } else {
+            op.write(path, "").await?;
+        }
```

Comment:
Perhaps I should have split it when providing the path above, instead of making the judgment here?

Reply:
Current style LGTM

Reply:
Right. Ok.